quarkchain/cluster/tests/test_cluster.py

```diff
@@ -2474,5 +2480,43 @@ def test_grpc_slave(self):
             self.assertEqual(
                 server.request_num, len(grpc_slaves),
             )
-            grpc_server1.stop(0)
-            grpc_server2.stop(0)
+            grpc_server1.stop(None)
+            grpc_server2.stop(None)
+
+    def test_grpc_server(self):
```

Comment:
seems the testing on grpc server is not directly related to cluster, and we only need a `RootState`. I suggest putting it into `test_grpc_server.py`, and creating a fake `RootState` as in `test_root_state.py`